hack/upload-release.sh

```diff
@@ -74,7 +74,7 @@ for os in $(echo linux darwin windows); do
   fi
 
   # Build
-  GOOS=${os} go build -o /tmp/${destination} ./cmd/kf
+  GOOS=${os} go build -o /tmp/${destination} "-X github.com/google/kf/pkg/kf/commands.Version=$(git rev-parse HEAD)" ./cmd/kf
```

Comment:
Just to be safe, we should check that git rev-parse succeeds and isn't the empty string